src/Objs/Redirect/RedirectComponent.js

```diff
@@ -2,37 +2,27 @@ import * as React from "react";
 import * as Scrivito from "scrivito";
 import { InPlaceEditingPlaceholder } from "../../Components/InPlaceEditingPlaceholder";
 
-class RedirectComponent extends React.Component {
-  componentDidMount() {
-    Scrivito.load(() => {
-      const link = this.props.page.get("link");
-      const url = link && Scrivito.urlFor(link);
+Scrivito.provideComponent("Redirect", ({ page }) => {
+  const link = page.get("link");
+  const url = link && Scrivito.urlFor(link);
```

Comment:
Please move the url resolving with a `load` inside of the `useEffect`. Otherwise its possible that the `useEffect` is triggered twice - first with the "unresolved" `urlFor(link)` and then with the resolved/loaded `urlFor(link)`.